rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+
+    # associations
+    book.rentals << self
+    person.rental << self
```

Comment:
- [ ] @SiansandaTamara great work implementing the changes requested, kindly fix your personal association to be plural rentals so that it achieves the 'belongs-to' association.